kirby/api/ext.py

```diff
@@ -1,61 +1,145 @@
+import logging
+import datetime
 import msgpack
 from kafka import KafkaConsumer, KafkaProducer
+from kafka.errors import NoBrokersAvailable
+
+logger = logging.getLogger(__name__)
+
+
+def retry(function):
+
+    NB_RETRY = 3
+
+    def function_decorated():
+        for counter in range(NB_RETRY):
+            try:
+                return_value = function_decorated()
+            except NoBrokersAvailable as e:
+                if counter == NB_RETRY:
```

Comment:
watchdog checks should always be done using `>=` comparisons, to ensure you don't end up in an endless loop

Reply:
Not used anymore